Hazelcast.Net/Hazelcast.Config/NearCacheConfig.cs

```diff
@@ -77,101 +83,198 @@ public virtual NearCacheConfigReadOnly GetAsReadOnly()
+        /// <li><c>LFU</c> (Least Frequently Used)</li>
+        /// <li><c>NONE</c> (no extra eviction, time-to-live-seconds or max-idle-seconds may still apply)</li>
+        /// <li><c>RANDOM</c> (random entry)</li>
+        /// </ul>
+        /// </summary>
+        /// <param name="evictionPolicy">eviction policy.</param>
+        /// <returns>this Near Cache config instance</returns>
         public virtual NearCacheConfig SetEvictionPolicy(string evictionPolicy)
         {
             _evictionPolicy = evictionPolicy;
             return this;
         }
 
+        /// <summary>
+        /// Sets the data type used to store entries.
+        /// <br/>
+        /// Possible values:
+        /// <ul>
+        /// <li><c>BINARY</c>: keys and values are stored as binary data</li>
+        /// <li><c>OBJECT</c>: values are stored in their object forms</li>
+        /// <li><c>NATIVE</c>: keys and values are stored in native memory</li>
+        /// </ul>
+        /// The default value is <c>BINARY</c>.
+        /// </summary>
+        /// <param name="inMemoryFormat">the data type used to store entries</param>
+        /// <returns>this Near Cache config instance</returns>
         public virtual NearCacheConfig SetInMemoryFormat(InMemoryFormat inMemoryFormat)
         {
             _inMemoryFormat = inMemoryFormat;
             return this;
         }
 
-        // this setter is for reflection based configuration building
+        /// <summary>
+        /// <see cref="SetInMemoryFormat(Hazelcast.Config.InMemoryFormat)"/>
+        /// this setter is for reflection based configuration building
+        /// </summary>
         public virtual NearCacheConfig SetInMemoryFormat(string inMemoryFormat)
         {
             Enum.TryParse(inMemoryFormat, true, out _inMemoryFormat);
             return this;
         }
 
+        /// <summary>
+        /// Sets if Near Cache entries are invalidated when the entries in the backing data structure are changed
+        /// (updated or removed).
+        /// <br/>
+        /// When this setting is enabled, a Hazelcast instance with a Near Cache listens for cluster-wide changes
+        /// on the entries of the backing data structure and invalidates its corresponding Near Cache entries.
+        /// Changes done on the local Hazelcast instance always invalidate the Near Cache immediately.
+        /// </summary>
+        /// <param name="invalidateOnChange"></param>
+        /// <returns>this Near Cache config instance</returns>
         public virtual NearCacheConfig SetInvalidateOnChange(bool invalidateOnChange)
         {
             _invalidateOnChange = invalidateOnChange;
             return this;
         }
 
+        /// <summary>
+        /// Set the maximum number of seconds each entry can stay in the Near Cache as untouched (not-read).
+        /// <br/>
+        /// Entries that are not read(touched) more than <c>maxIdleSeconds</c> value will get removed from the Near Cache.
```

Comment:
I think it should be like: /// Entries that are not read (untouched) more than...

Or just like this: /// Entries that are not read more than...